edx_reverification_block/xblock/reverification_block.py

```diff
@@ -249,3 +251,23 @@ def _render_template(template_path, context):
         template_str = ReverificationBlock._resource(template_path)
         template = Template(template_str)
         return template.render(Context(context))
+
+    def validate(self):
+        """
+        Basic validation for this XBlock.
+        """
+        reverifcation_block_validation = super(ReverificationBlock, self).validate()
+        if not isinstance(reverifcation_block_validation, Validation):
+            reverifcation_block_validation = Validation.copy(reverifcation_block_validation)
+
+        if self.is_configured:
+            reverifcation_block_validation.add(
+                ValidationMessage(
+                    ValidationMessage.WARNING,
+                    _(
```

Comment:
For i18n in XBlocks, we need to use the `i18n` service.  This is how we used it in ORA2:
https://github.com/edx/edx-ora2/blob/master/openassessment/xblock/openassessmentblock.py#L456-L459